src/spearman.jl

```diff
@@ -27,7 +27,7 @@ export CorrelationTest, SpearmanCorrelationTest
 abstract CorrelationTest <: HypothesisTest
 
 "Sum squared difference of ranks (midranks for ties)"
-spearman_S(xrank, yrank) = sumabs2(xrank .- yrank)
+spearman_S(xrank,yrank) = sumabs2(xrank .- yrank)
```

Comment:
The style convention in this repo seems to have spaces between arguments and items in a collection. The inconsistent padding I was referring to was like `somefunction(     arg1, arg2     )` and `somearray = [      1, 2, 3           ]` rather than the `somefunction(arg1, arg2)` and `somearray = [1, 2, 3]` convention I observe to be more common in this repo (and personally preferred, but who cares about that). I'm not a committer in this repo, but I do appreciate having consistent style in a codebase.